src/aavrutiDJun20/ReverseIntNumber.java

```diff
@@ -0,0 +1,20 @@
+package aavrutiDJun20;
+
+public class ReverseIntNumber {
+	//Print reverse of the given Number
+	void reverseNumber(int number) {
+		int num=number;
+		int reverseNumber = 0;
+		System.out.print("Reverse Number of " + number + " is --> " );
+		while(num>0) {
+			reverseNumber = num%10;
+			num = num/10;
+			System.out.print(reverseNumber);
```

Comment:
@AavrutiDJun20 ,
here integer is not reversed exactly - it is chopped digit by digit and printed in reverse order.
Please implement integer reverse logic - which is just a small extension to your login.